api/.env.development

```diff
@@ -0,0 +1 @@
+engine_connect_args = {"check_same_thread": false}
```

Comment:
```suggestion
engine_connect_args = {"check_same_thread": False}
```
Dont know if we want python-style here, but if no just ignore this comment and merge if you want